src/http/client/response.cr

```diff
@@ -31,6 +31,10 @@ class HTTP::Client::Response
     @body
   end
 
+  def successful?
```

Comment:
Can we name it `success?` instead? Shorter and is similar to response handling on things like Faraday.